packages/libs/react-components/src/components/Input/styles.ts

```diff
@@ -0,0 +1,130 @@
+/* eslint @kadena-dev/typedef-var: 0 */
+// TODO: Remove this when this issue is resolved: https://github.com/kadena-community/kadena.js/issues/201
+
+import { styled } from '../../styles';
+
+export const styleVariant = {} as const;
+
+export const StyledInputGroupWrapper = styled('div', {
+  color: '$colors$neutral6',
+  '&.disabled': {
+    color: '$colors$neutral3',
+  },
+});
+
+export const StyledInputGroupHeader = styled('div', {
+  display: 'flex',
+  alignItems: 'center',
+  paddingBottom: '8px',
+  '*': {
+    display: 'inline-block',
+  },
+});
+
+export const StyledLabel = styled('label', {
+  fontWeight: '500',
+  fontSize: '14px',
+});
```

Comment:
I think it would be beneficial to update the `Text` component to accept `label` as a variant and element and define this style there as I expect this styling will be reused in other places as well